bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -331,6 +333,15 @@ spec:
           {{- if .Values.pgpool.resources }}
           resources: {{- toYaml .Values.pgpool.resources | nindent 12 }}
           {{- end }}
+          {{- if or
+            (or .Values.pgpool.configuration .Values.pgpool.configurationCM)
+            (or .Values.pgpool.initdbScripts .Values.pgpool.initdbScriptsCM)
+            (.Values.pgpool.initdbScriptsSecret)
+            (.Values.postgresql.usePasswordFile)
+            (.Values.pgpool.usePasswordFile)
+            (.Values.pgpool.tls.enabled)
+            (.Values.pgpool.extraVolumeMounts)
+          }}
```

Comment:
Could you please not split this into multiple lines?
I know that a single super-long line isn't pretty or precisely easy to read, but at the moment I prefer sticking with the style used in the rest of the catalog.
In the future, we may implement some linting guidelines and extend it to all the charts in the catalog, including here.